dashboard/pubspec.lock

```diff
@@ -268,6 +268,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "4.0.0"
+  icon_plugin:
```

Comment:
Unfortunately, we can't pull this into this project if it's licensed with GNU GPL v3

Reply:
As a disclaimer, I'm not a lawyer :-)

Most popular open source licenses will work, this is one of the few that prevents us from including it. You could probably swipe through our existing pubspec to see, but I think the majority are MIT.

Reply:
hahaha thought you are! All the plugins I looked at are made by flutter team :D. I'll take a look again in few.

Reply:
https://pub.dev/packages/github is an example we use a lot in Cocoon.